examples/workflow_example_cep.jl

```diff
@@ -20,41 +20,41 @@ ts_seg_data = run_clust(ts_input_data;method="kmeans",representation="centroid",
 optimizer=Clp.Optimizer
 
 # tweak the CO2 level
-co2_result = run_opt(ts_clust_data.clust_data,cep_data,optimizer;co2_limit=50) #generally values between 1250 and 10 are interesting
+co2_result = run_opt(ts_clust_data.clust_data,cep_data,optimizer;limit_emission=Dict{String,Number}("CO2/electricity"=>50)) #generally values between 1250 and 10 are interesting
```

Comment:
What does 'CO2/electricity' mean? How about `kg_CO2/kWh` or `CO2:kg/kWh`? I feel having units somewhere could make things helpful. Similar for lost load.